experiments/scripts/delete_old_parquet.py

```diff
@@ -0,0 +1,58 @@
+from google.cloud import storage
+from pprint import pprint
```

Comment:
Did you run `flake8` on this? I'm pretty sure you're not using this and can remove it.

Reply:
I set up `flake8` not to cover the `experiments` folder because I was having to edit all the issues in files that were only for experimentation (i.e., weren't associated with source code) before committing them. To cut down on this, I listed `experiments` under the `exclude` argument in `.flake8`.